utils/detect_sound.py

```
@@ -0,0 +1,55 @@
+        processes = set()
+        for line in output:
+            parts = line.split(" ", 1)
+            if len(parts) > 1:
+                # Get the command name (basename)
+                cmd = parts[1].split()[0]
+                processes.add(os.path.basename(cmd))
+        return processes
+    except Exception:
+        return set()
+
+def detect_sound_server(uid, home):
```

Ruff (0.14.10)

Unused function argument: `home`

(ARG001)